app/Models/Signup.php

```diff
@@ -146,6 +146,9 @@ public function toBlinkPayload()
         // Blink expects quantity to be a number.
         $quantity = $this->quantity === null ? 0 : $this->quantity;
 
+        // Array values aren't supported in Customer.io event attributes.
+        $campaign_cause = implode(',', (array) $this->campaign['cause']);
```

Comment:
```suggestion
        $campaign_cause = (string) optional($this->campaign)->getAttributes()['cause'];
```

See https://github.com/DoSomething/rogue/pull/896#discussion_r303555913